src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -0,0 +1,17 @@
+package lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class WinningNumbers extends Lotto {
+
+    private List<LottoNumber> winningNumbers;
+
+    public WinningNumbers(String winningNumbersString) {
+        super(Arrays.asList(winningNumbersString.split(",")).stream()
+                .map(winningNumberString -> new LottoNumber(winningNumberString))
+                .collect(Collectors.toList()));
```

Comment:
```suggestion
        super(Arrays.stream(winningNumbersString.split(","))
                    .map(LottoNumber::new)
                    .collect(Collectors.toList()));
```
`Arrays.asList` 로 `List`를 생성하고 만들기보다는 `Arrays.stream(arr[])` 을 바로 활용하면 공간 낭비를 줄일 수 있어요.
두 번째 줄은 람다로 변경이 가능합니다.

그리고 가급적이면 파싱의 책임은 분리해주는게 좋습니다.
예상 밖의 문자가 들어올 수 있어서 애초에 만들 때 제대로 된 걸 받아서 만드는걸 추천드려요.

Reply:
Arrays.stream(arr[]), 람다 수정반영하겠습니다!
파싱의 책임은 분리는.. StringSplitter인터페이스를 만들고 책임을 다른 클래스로 이관해서 작업해보겠습니다. 
다음리뷰때 피드백 부탁드리겠습니다.

감사합니다!